tools/admin/src/main/scala/whisk/core/database/LimitsCommand.scala

```diff
@@ -64,11 +64,27 @@ class LimitsCommand extends Subcommand("limits") with WhiskCommand {
         validate = _ >= 0,
         name = "concurrentInvocations",
         noshort = true)
+    val allowedKinds =
+      opt[List[String]](
+        descr = "list of runtime kinds allowed in this namespace",
+        argName = "ALLOWEDKINDS",
+        name = "allowedKinds",
+        noshort = true,
+        default = None)
+
+    lazy val allowedKindSet = allowedKinds.toOption match {
+      case None       => None
+      case Some(list) => Some(list.toSet)
+    }
 
     lazy val limits: LimitEntity =
       new LimitEntity(
         EntityName(namespace()),
-        UserLimits(invocationsPerMinute.toOption, firesPerMinute.toOption, concurrentInvocations.toOption))
+        UserLimits(
+          invocationsPerMinute.toOption,
+          firesPerMinute.toOption,
+          concurrentInvocations.toOption,
+          allowedKindSet))
```

Comment:
Instead of introducing `allowedKindSet` you can use [allowedKind.toSet][1]

[1]: https://scalafiddle.io/sf/my792Nt/0

Reply:
Thanks @chetanmeh, However it doesn't appear that `ScallopOption[List[String]]` has a `toSet` method whic was the reason for the explicit conversion. Am I missing something?
```> Task :tools:admin:compileScala
/wrk/runtime/dev/openwhisk/tools/admin/src/main/scala/whisk/core/database/LimitsCommand.scala:82: value toSet is not a member of org.rogach.scallop.ScallopOption[List[String]]
          allowedKinds.toSet))
                       ^
one error found

> Task :tools:admin:compileScala FAILED
```

Reply:
You can map the option via `allowedKinds.toOption.map(_.toSet)`